### ISSUE TITLE
jupyterhub-singleuser fails when user's home directory doesn't exist

### ISSUE THREAD

**Issue by mario-campos** [jupyterhub/jupyterhub#1503]
**Summary**

Our JupyterHub server is using SSSD to link authentication and authorization to Active Directory. With SSSD, a user's home directory won't be created until/unless that user SSHs into the system. So, in essence, we have JupyterHub users (that don't use SSH, or even really know what it is) try to log into JupyterHub, but they get all sorts of internal-server errors. The source of the error seems to be that `jupyterhub-singleuser` will ensure that `$HOME` exists by creating it if it does not. However, after having `setuid()`, it lacks the privilege to create `$HOME`.

I think it shouldn't attempt to create `$HOME`. Our users don't necessarily have `$HOME` directories that exist, and that's fine—they don't need them.

Below, I execute `jupyterhub-singleuser` myself, but I am simply trying to display the errors that get eaten up by JupyterHub when it attempts to spawn `jupyterhub-singleuser`.

**How to reproduce the issue**

```
$ sudo useradd -M foouser
$ ls /home
mario
$ sudo -u foouser /mnt/ds/anaconda/bin/jupyterhub-singleuser --port=57269 --notebook-dir=/mnt/ds/notebooks --debug
```

**What you expected to happen**

```
$ sudo -u foouser /mnt/ds/anaconda3/bin/jupyterhub-singleuser --port=57269 --notebook-dir=/mnt/ds/notebooks --debug
/mnt/ds/anaconda3/lib/python3.6/site-packages/IPython/paths.py:68: UserWarning: IPython parent '/home/foouser' is not a writable location, using a temp directory.
  " using a temp directory.".format(parent))
...
[D 2017-10-25 16:22:57.927 SingleUserNotebookApp application:555] Looking for jupyter_notebook_config in /etc/jupyter
[D 2017-10-25 16:22:57.927 SingleUserNotebookApp application:555] Looking for jupyter_notebook_config in /usr/local/etc/jupyter
[D 2017-10-25 16:22:57.927 SingleUserNotebookApp application:555] Looking for jupyter_notebook_config in /mnt/ds/anaconda3/etc/jupyter
[D 2017-10-25 16:22:57.927 SingleUserNotebookApp application:577] Loaded config file: /mnt/ds/anaconda3/etc/jupyter/jupyter_notebook_config.json
[D 2017-10-25 16:22:57.932 SingleUserNotebookApp application:647] Exiting application: jupyter-notebook
```

**What actually happens**

```
$ sudo -u foouser /mnt/ds/anaconda/bin/jupyterhub-singleuser --port=57269 --notebook-dir=/mnt/ds/notebooks --debug
/mnt/ds/anaconda3/lib/python3.6/site-packages/IPython/paths.py:68: UserWarning: IPython parent '/home/foouser' is not a writable location, using a temp directory.
  " using a temp directory.".format(parent))
Traceback (most recent call last):
  File "/mnt/ds/anaconda3/bin/jupyterhub-singleuser", line 6, in <module>
    main()
  File "/mnt/ds/anaconda3/lib/python3.6/site-packages/jupyterhub/singleuser.py", line 455, in main
    return SingleUserNotebookApp.launch_instance(argv)
  File "/mnt/ds/anaconda3/lib/python3.6/site-packages/jupyter_core/application.py", line 267, in launch_instance
    return super(JupyterApp, cls).launch_instance(argv=argv, **kwargs)
  File "/mnt/ds/anaconda3/lib/python3.6/site-packages/traitlets/config/application.py", line 657, in launch_instance
    app.initialize(argv)
  File "<decorator-gen-7>", line 2, in initialize
  File "/mnt/ds/anaconda3/lib/python3.6/site-packages/traitlets/config/application.py", line 87, in catch_config_error
    return method(app, *args, **kwargs)
  File "/mnt/ds/anaconda3/lib/python3.6/site-packages/notebook/notebookapp.py", line 1290, in initialize
    super(NotebookApp, self).initialize(argv)
  File "<decorator-gen-6>", line 2, in initialize
  File "/mnt/ds/anaconda3/lib/python3.6/site-packages/traitlets/config/application.py", line 87, in catch_config_error
    return method(app, *args, **kwargs)
  File "/mnt/ds/anaconda3/lib/python3.6/site-packages/jupyter_core/application.py", line 243, in initialize
    self.migrate_config()
  File "/mnt/ds/anaconda3/lib/python3.6/site-packages/jupyterhub/singleuser.py", line 314, in migrate_config
    super(SingleUserNotebookApp, self).migrate_config()
  File "/mnt/ds/anaconda3/lib/python3.6/site-packages/jupyter_core/application.py", line 169, in migrate_config
    migrate()
  File "/mnt/ds/anaconda3/lib/python3.6/site-packages/jupyter_core/migrate.py", line 240, in migrate
    ensure_dir_exists(env['jupyter_config'])
  File "/mnt/ds/anaconda3/lib/python3.6/site-packages/ipython_genutils/path.py", line 167, in ensure_dir_exists
    os.makedirs(path, mode=mode)
  File "/mnt/ds/anaconda3/lib/python3.6/os.py", line 210, in makedirs
    makedirs(head, mode, exist_ok)
  File "/mnt/ds/anaconda3/lib/python3.6/os.py", line 220, in makedirs
    mkdir(name, mode)
PermissionError: [Errno 13] Permission denied: '/home/foouser'
```

**Share what version of JupyterHub you are using**

Running `jupyter troubleshoot` from the command line, if possible, and posting
its output would also be helpful.

```
$ /mnt/ds/anaconda3/bin/jupyter troubleshoot
DEPRECATION: The default format will switch to columns in the future. You can use --format=(legacy|columns) (or define a format=(legacy|columns) in your pip.conf under the [list] section) to disable this warning.
$PATH:
        /mnt/ds/anaconda3/bin
        /usr/local/bin
        /usr/bin
        /usr/local/sbin
        /usr/sbin
        /home/mario/.local/bin
        /home/mario/bin

sys.path:
        /mnt/ds/anaconda3/bin
        /mnt/ds/anaconda3/lib/python36.zip
        /mnt/ds/anaconda3/lib/python3.6
        /mnt/ds/anaconda3/lib/python3.6/lib-dynload
        /mnt/ds/anaconda3/lib/python3.6/site-packages
        /mnt/ds/anaconda3/lib/python3.6/site-packages/Mako-1.0.7-py3.6.egg

sys.executable:
        /mnt/ds/anaconda3/bin/python

sys.version:
        3.6.2 |Anaconda, Inc.| (default, Sep 30 2017, 18:42:57)
        [GCC 7.2.0]

platform.platform():
        Linux-3.10.0-693.5.2.el7.x86_64-x86_64-with-redhat-7.4-Maipo

which -a jupyter:
        /mnt/ds/anaconda3/bin/jupyter

pip list:
        alabaster (0.7.10)
        alembic (0.9.6)
        anaconda-client (1.6.5)
        anaconda-navigator (1.6.8)
        anaconda-project (0.8.0)
        asn1crypto (0.22.0)
        astroid (1.5.3)
        astropy (2.0.2)
        Babel (2.5.0)
        backports.shutil-get-terminal-size (1.0.0)
        beautifulsoup4 (4.6.0)
        bitarray (0.8.1)
        bkcharts (0.2)
        blaze (0.11.3)
        bleach (2.0.0)
        bokeh (0.12.7)
        boto (2.48.0)
        Bottleneck (1.2.1)
        certifi (2017.7.27.1)
        cffi (1.10.0)
        chardet (3.0.4)
        click (6.7)
        cloudpickle (0.4.0)
        clyent (1.2.2)
        colorama (0.3.9)
        conda (4.3.29)
        conda-build (3.0.23)
        conda-verify (2.0.0)
        contextlib2 (0.5.5)
        cryptography (2.0.3)
        cycler (0.10.0)
        Cython (0.26.1)
        cytoolz (0.8.2)
        dask (0.15.2)
        datashape (0.5.4)
        decorator (4.1.2)
        distributed (1.18.3)
        docutils (0.14)
        entrypoints (0.2.3)
        et-xmlfile (1.0.1)
        fastcache (1.0.2)
        filelock (2.0.12)
        Flask (0.12.2)
        Flask-Cors (3.0.3)
        gevent (1.2.2)
        glob2 (0.5)
        gmpy2 (2.0.8)
        greenlet (0.4.12)
        h5py (2.7.0)
        heapdict (1.0.0)
        html5lib (0.999999999)
        idna (2.6)
        imageio (2.2.0)
        imagesize (0.7.1)
        ipykernel (4.6.1)
        ipython (6.1.0)
        ipython-genutils (0.2.0)
        ipywidgets (7.0.0)
        isort (4.2.15)
        itsdangerous (0.24)
        jdcal (1.3)
        jedi (0.10.2)
        Jinja2 (2.9.6)
        jsonschema (2.6.0)
        jupyter-client (5.1.0)
        jupyter-console (5.2.0)
        jupyter-core (4.3.0)
        jupyterhub (0.8.0)
        jupyterlab (0.27.0)
        jupyterlab-launcher (0.4.0)
        lazy-object-proxy (1.3.1)
        llvmlite (0.20.0)
        locket (0.2.0)
        lxml (3.8.0)
        Mako (1.0.7)
        MarkupSafe (1.0)
        matplotlib (2.0.2)
        mccabe (0.6.1)
        mistune (0.7.4)
        mpmath (0.19)
        msgpack-python (0.4.8)
        multipledispatch (0.4.9)
        navigator-updater (0.1.0)
        nbconvert (5.3.1)
        nbformat (4.4.0)
        networkx (1.11)
        nltk (3.2.4)
        nose (1.3.7)
        notebook (5.0.0)
        numba (0.35.0+10.g143f70e90)
        numexpr (2.6.2)
        numpy (1.13.1)
        numpydoc (0.7.0)
        odo (0.5.1)
        olefile (0.44)
        openpyxl (2.4.8)
        packaging (16.8)
        pamela (0.3.0)
        pandas (0.20.3)
        pandocfilters (1.4.2)
        partd (0.3.8)
        path.py (10.3.1)
        pathlib2 (2.3.0)
        patsy (0.4.1)
        pep8 (1.7.0)
        pexpect (4.2.1)
        pickleshare (0.7.4)
        Pillow (4.2.1)
        pip (9.0.1)
        pkginfo (1.4.1)
        ply (3.10)
        prompt-toolkit (1.0.15)
        psutil (5.2.2)
        ptyprocess (0.5.2)
        py (1.4.34)
        pycodestyle (2.3.1)
        pycosat (0.6.2)
        pycparser (2.18)
        pycrypto (2.6.1)
        pycurl (7.43.0)
        pyflakes (1.5.0)
        Pygments (2.2.0)
        pylint (1.7.2)
        pyodbc (4.0.17)
        pyOpenSSL (17.2.0)
        pyparsing (2.2.0)
        PySocks (1.6.7)
        pytest (3.2.1)
        python-dateutil (2.6.1)
        python-editor (1.0.3)
        python-oauth2 (1.0.1)
        pytz (2017.2)
        PyWavelets (0.5.2)
        PyYAML (3.12)
        pyzmq (16.0.2)
        QtAwesome (0.4.4)
        qtconsole (4.3.1)
        QtPy (1.3.1)
        requests (2.18.4)
        rope (0.10.5)
        ruamel-yaml (0.11.14)
        scikit-image (0.13.0)
        scikit-learn (0.19.0)
        scipy (0.19.1)
        seaborn (0.8)
        setuptools (36.5.0.post20170921)
        simplegeneric (0.8.1)
        singledispatch (3.4.0.3)
        six (1.10.0)
        snowballstemmer (1.2.1)
        sortedcollections (0.5.3)
        sortedcontainers (1.5.7)
        Sphinx (1.6.3)
        sphinxcontrib-websupport (1.0.1)
        spyder (3.2.3)
        SQLAlchemy (1.1.13)
        statsmodels (0.8.0)
        sympy (1.1.1)
        tables (3.4.2)
        tblib (1.3.2)
        terminado (0.6)
        testpath (0.3.1)
        toolz (0.8.2)
        tornado (4.5.2)
        traitlets (4.3.2)
        typing (3.6.2)
        unicodecsv (0.14.1)
        urllib3 (1.22)
        wcwidth (0.1.7)
        webencodings (0.5.1)
        Werkzeug (0.12.2)
        wheel (0.29.0)
        widgetsnbextension (3.0.2)
        wrapt (1.10.11)
        xlrd (1.1.0)
        XlsxWriter (0.9.8)
        xlwt (1.3.0)
        zict (0.1.2)

conda list:
        # packages in environment at /mnt/ds/anaconda3:
        #
        _ipyw_jlab_nb_ext_conf    0.1.0            py36he11e457_0
        alabaster                 0.7.10           py36h306e16b_0
        alembic                   0.9.6                      py_0    conda-forge
        anaconda                  5.0.0.1          py36hfb0b618_1
        anaconda-client           1.6.5            py36h19c0dcd_0
        anaconda-navigator        1.6.8            py36h672ccc7_0
        anaconda-project          0.8.0            py36h29abdf5_0
        asn1crypto                0.22.0           py36h265ca7c_1
        astroid                   1.5.3            py36hbdb9df2_0
        astropy                   2.0.2            py36ha51211e_4
        babel                     2.5.0            py36h7d14adf_0
        backports                 1.0              py36hfa02d7e_1
        backports.shutil_get_terminal_size 1.0.0            py36hfea85ff_2
        beautifulsoup4            4.6.0            py36h49b8c8c_1
        bitarray                  0.8.1            py36h5834eb8_0
        bkcharts                  0.2              py36h735825a_0
        blaze                     0.11.3           py36h4e06776_0
        bleach                    2.0.0            py36h688b259_0
        bokeh                     0.12.7           py36h169c5fd_1
        boto                      2.48.0           py36h6e4cd66_1
        bottleneck                1.2.1            py36haac1ea0_0
        ca-certificates           2017.08.26           h1d4fec5_0
        cairo                     1.14.10              h58b644b_4
        certifi                   2017.7.27.1      py36h8b7b77e_0
        cffi                      1.10.0           py36had8d393_1
        chardet                   3.0.4            py36h0f667ec_1
        click                     6.7              py36h5253387_0
        cloudpickle               0.4.0            py36h30f8c20_0
        clyent                    1.2.2            py36h7e57e65_1
        colorama                  0.3.9            py36h489cec4_0
        conda                     4.3.29                   py36_0    conda-forge
        conda-build               3.0.23           py36h1da9bb0_0
        conda-env                 2.6.0                         0    conda-forge
        conda-verify              2.0.0            py36h98955d8_0
        configurable-http-proxy   3.0.0                         0    conda-forge
        contextlib2               0.5.5            py36h6c84a62_0
        cryptography              2.0.3            py36ha225213_1
        curl                      7.55.1               hcb0b314_2
        cycler                    0.10.0           py36h93f1223_0
        cython                    0.26.1           py36h21c49d0_0
        cytoolz                   0.8.2            py36h708bfd4_0
        dask                      0.15.2           py36h9b48dc4_0
        dask-core                 0.15.2           py36h0f988a8_0
        datashape                 0.5.4            py36h3ad6b5c_0
        dbus                      1.10.22              h3b5a359_0
        decorator                 4.1.2            py36hd076ac8_0
        distributed               1.18.3           py36h73cd4ae_0
        docutils                  0.14             py36hb0f60f5_0
        entrypoints               0.2.3            py36h1aec115_2
        et_xmlfile                1.0.1            py36hd6bccc3_0
        expat                     2.2.4                hc00ebd1_1
        fastcache                 1.0.2            py36h5b0c431_0
        filelock                  2.0.12           py36hacfa1f5_0
        flask                     0.12.2           py36hb24657c_0
        flask-cors                3.0.3            py36h2d857d3_0
        fontconfig                2.12.4               h88586e7_1
        freetype                  2.8                  h52ed37b_0
        get_terminal_size         1.0.0                haa9412d_0
        gevent                    1.2.2            py36h2fe25dc_0
        glib                      2.53.6               hc861d11_1
        glob2                     0.5              py36h2c1b292_1
        gmp                       6.1.2                hb3b607b_0
        gmpy2                     2.0.8            py36h55090d7_1
        graphite2                 1.3.10               hc526e54_0
        greenlet                  0.4.12           py36h2d503a6_0
        gst-plugins-base          1.12.2               he3457e5_0
        gstreamer                 1.12.2               h4f93127_0
        h5py                      2.7.0            py36he81ebca_1
        harfbuzz                  1.5.0                h2545bd6_0
        hdf5                      1.10.1               hb0523eb_0
        heapdict                  1.0.0            py36h79797d7_0
        html5lib                  0.999999999      py36h2cfc398_0
        icu                       58.2                 h211956c_0
        idna                      2.6              py36h82fb2a8_1
        imageio                   2.2.0            py36he555465_0
        imagesize                 0.7.1            py36h52d8127_0
        intel-openmp              2018.0.0             h15fc484_7
        ipykernel                 4.6.1            py36hbf841aa_0
        ipython                   6.1.0            py36hc72a948_1
        ipython_genutils          0.2.0            py36hb52b0d5_0
        ipywidgets                7.0.0            py36h7b55c3a_0
        isort                     4.2.15           py36had401c0_0
        itsdangerous              0.24             py36h93cc618_1
        jbig                      2.1                  hdba287a_0
        jdcal                     1.3              py36h4c697fb_0
        jedi                      0.10.2           py36h552def0_0
        jinja2                    2.9.6            py36h489bce4_1
        jpeg                      9b                   habf39ab_1
        jsonschema                2.6.0            py36h006f8b5_0
        jupyter                   1.0.0            py36h9896ce5_0
        jupyter_client            5.1.0            py36h614e9ea_0
        jupyter_console           5.2.0            py36he59e554_1
        jupyter_core              4.3.0            py36h357a921_0
        jupyterhub                0.8.0                    py36_0    conda-forge
        jupyterlab                0.27.0           py36h86377d0_2
        jupyterlab_launcher       0.4.0            py36h4d8058d_0
        lazy-object-proxy         1.3.1            py36h10fcdad_0
        libedit                   3.1                  heed3624_0
        libffi                    3.2.1                h4deb6c0_3
        libgcc                    7.2.0                h69d50b8_2
        libgcc-ng                 7.2.0                hcbc56d2_1
        libgfortran-ng            7.2.0                h6fcbd8e_1
        libpng                    1.6.32               hda9c8bc_2
        libsodium                 1.0.13               h31c71d8_2
        libssh2                   1.8.0                h8c220ad_2
        libstdcxx-ng              7.2.0                h24385c6_1
        libtiff                   4.0.8                h90200ff_9
        libtool                   2.4.6                hd50d1a6_0
        libxcb                    1.12                 he6ee5dd_2
        libxml2                   2.9.4                h6b072ca_5
        libxslt                   1.1.29               hcf9102b_5
        llvmlite                  0.20.0                   py36_0
        locket                    0.2.0            py36h787c0ad_1
        lxml                      3.8.0            py36h6c6e760_0
        lzo                       2.10                 hc0eb8fc_0
        mako                      1.0.7                    py36_0    conda-forge
        markupsafe                1.0              py36hd9260cd_1
        matplotlib                2.0.2            py36h2acb4ad_1
        mccabe                    0.6.1            py36h5ad9710_1
        mistune                   0.7.4            py36hbab8784_0
        mkl                       2018.0.0             hb491cac_4
        mkl-service               1.1.2            py36h17a0993_4
        mpc                       1.0.3                hf803216_4
        mpfr                      3.1.5                h12ff648_1
        mpmath                    0.19             py36h8cc018b_2
        msgpack-python            0.4.8            py36hec4c5d1_0
        multipledispatch          0.4.9            py36h41da3fb_0
        navigator-updater         0.1.0            py36h14770f7_0
        nbconvert                 5.3.1            py36hb41ffb7_0
        nbformat                  4.4.0            py36h31c9010_0
        ncurses                   6.0                  h06874d7_1
        networkx                  1.11             py36hfb3574a_0
        nltk                      3.2.4            py36h1a0979f_0
        nodejs                    6.11.0                        0    conda-forge
        nose                      1.3.7            py36hcdf7029_2
        notebook                  5.0.0            py36h0b20546_2
        numba                     0.35.0             np113py36_10
        numexpr                   2.6.2            py36hdd3393f_1
        numpy                     1.13.1           py36h5bc529a_2
        numpydoc                  0.7.0            py36h18f165f_0
        odo                       0.5.1            py36h90ed295_0
        olefile                   0.44             py36h79f9f78_0
        openpyxl                  2.4.8            py36h41dd2a8_1
        openssl                   1.0.2l               h9d1a558_3
        packaging                 16.8             py36ha668100_1
        pamela                    0.3.0                    py36_0    conda-forge
        pandas                    0.20.3           py36h842e28d_2
        pandoc                    1.19.2.1             hea2e7c5_1
        pandocfilters             1.4.2            py36ha6701b7_1
        pango                     1.40.11              hedb6d6b_0
        partd                     0.3.8            py36h36fd896_0
        patchelf                  0.9                  hf79760b_2
        path.py                   10.3.1           py36he0c6f6d_0
        pathlib2                  2.3.0            py36h49efa8e_0
        patsy                     0.4.1            py36ha3be15e_0
        pcre                      8.41                 hc71a17e_0
        pep8                      1.7.0            py36h26ade29_0
        pexpect                   4.2.1            py36h3b9d41b_0
        pickleshare               0.7.4            py36h63277f8_0
        pillow                    4.2.1            py36h9119f52_0
        pip                       9.0.1            py36h30f8307_2
        pixman                    0.34.0               ha72d70b_1
        pkginfo                   1.4.1            py36h215d178_1
        ply                       3.10             py36hed35086_0
        prompt_toolkit            1.0.15           py36h17d85b1_0
        psutil                    5.2.2            py36h74c8701_0
        ptyprocess                0.5.2            py36h69acd42_0
        py                        1.4.34           py36h0712aa3_1
        pycodestyle               2.3.1            py36hf609f19_0
        pycosat                   0.6.2            py36h1a0ea17_1
        pycparser                 2.18             py36hf9f622e_1
        pycrypto                  2.6.1            py36h6998063_1
        pycurl                    7.43.0           py36h5e72054_3
        pyflakes                  1.5.0            py36h5510808_1
        pygments                  2.2.0            py36h0d3125c_0
        pylint                    1.7.2            py36h484ab97_0
        pyodbc                    4.0.17           py36h999153c_0
        pyopenssl                 17.2.0           py36h5cc804b_0
        pyparsing                 2.2.0            py36hee85983_1
        pyqt                      5.6.0            py36h0386399_5
        pysocks                   1.6.7            py36hd97a5b1_1
        pytables                  3.4.2            py36hdce54c9_1
        pytest                    3.2.1            py36h11ad3bb_1
        python                    3.6.2               hdfe5801_15
        python-dateutil           2.6.1            py36h88d3b88_1
        python-editor             1.0.3                    py36_0    conda-forge
        python-oauth2             1.0.1                    py36_0    conda-forge
        pytz                      2017.2           py36hc2ccc2a_1
        pywavelets                0.5.2            py36he602eb0_0
        pyyaml                    3.12             py36hafb9ca4_1
        pyzmq                     16.0.2           py36h3b0cf96_2
        qt                        5.6.2               h974d657_12
        qtawesome                 0.4.4            py36h609ed8c_0
        qtconsole                 4.3.1            py36h8f73b5b_0
        qtpy                      1.3.1            py36h3691cc8_0
        readline                  7.0                  hac23ff0_3
        requests                  2.18.4           py36he2e5f8d_1
        rope                      0.10.5           py36h1f8c17e_0
        ruamel_yaml               0.11.14          py36ha2fb22d_2
        scikit-image              0.13.0           py36had3c07a_1
        scikit-learn              0.19.0           py36h97ac459_2
        scipy                     0.19.1           py36h9976243_3
        seaborn                   0.8.0            py36h197244f_0
        setuptools                36.5.0           py36he42e2e1_0
        simplegeneric             0.8.1            py36h2cb9092_0
        singledispatch            3.4.0.3          py36h7a266c3_0
        sip                       4.18.1           py36h51ed4ed_2
        six                       1.10.0           py36hcac75e4_1
        snowballstemmer           1.2.1            py36h6febd40_0
        sortedcollections         0.5.3            py36h3c761f9_0
        sortedcontainers          1.5.7            py36hdf89491_0
        sphinx                    1.6.3            py36he5f0bdb_0
        sphinxcontrib             1.0              py36h6d0f590_1
        sphinxcontrib-websupport  1.0.1            py36hb5cb234_1
        spyder                    3.2.3            py36he38cbf7_1
        sqlalchemy                1.1.13           py36hfb5efd7_0
        sqlite                    3.20.1               h6d8b0f3_1
        statsmodels               0.8.0            py36h8533d0b_0
        sympy                     1.1.1            py36hc6d1c1c_0
        tblib                     1.3.2            py36h34cf8b6_0
        terminado                 0.6              py36ha25a19f_0
        testpath                  0.3.1            py36h8cadb63_0
        tk                        8.6.7                h5979e9b_1
        toolz                     0.8.2            py36h81f2dff_0
        tornado                   4.5.2            py36h1283b2a_0
        traitlets                 4.3.2            py36h674d592_0
        typing                    3.6.2            py36h7da032a_0
        unicodecsv                0.14.1           py36ha668878_0
        unixodbc                  2.3.4                hc36303a_1
        urllib3                   1.22             py36hbe7ace6_0
        wcwidth                   0.1.7            py36hdf4376a_0
        webencodings              0.5.1            py36h800622e_1
        werkzeug                  0.12.2           py36hc703753_0
        wheel                     0.29.0           py36he7f4e38_1
        widgetsnbextension        3.0.2            py36hd01bb71_1
        wrapt                     1.10.11          py36h28b7045_0
        xlrd                      1.1.0            py36h1db9f0c_1
        xlsxwriter                0.9.8            py36hf41c223_0
        xlwt                      1.3.0            py36h7b00a1f_0
        xz                        5.2.3                h2bcbf08_1
        yaml                      0.1.7                h96e3832_1
        zeromq                    4.2.2                hb0b69da_1
        zict                      0.1.2            py36ha0d441b_0
        zlib                      1.2.11               hfbfcf68_1
```

**Comment by JoaoBarreto** [comment]
Did you find the solution for this? I'm currently having the same problem... :(

**Comment by mario-campos** [comment]
It was solved in issue #118.
